packages/components/src/components/circular-progress/circular-progress.tsx

```diff
@@ -25,7 +25,7 @@ const CircularProgress = ({
 }: TCircularProgressProps) => {
     const normalizedRadius = radius - stroke / 2;
     const circumference = normalizedRadius * 2 * Math.PI;
-    const strokeDashoffset = circumference - (progress / 100) * circumference;
+    const strokeDashoffset = circumference - (progress / 100) * circumference ?? 0;
```

Comment:
There were cases when `circumference - (progress / 100) * circumference === null `or `undefined`? I thought it is number.
Please, ignore if not relevant

Reply:
There was a case it return NaN and it throws error

Reply:
circumference should not be null as per the above calc

Reply:
It was throwing the error i mentioned above before, but i couldnt reproduce it anymore. Will revert it 🙏

Reply:
![image](https://github.com/binary-com/deriv-app/assets/125247833/9048439e-82a3-48c4-98e2-8a202760c1cb)
i can reproduce the error. it only happen when resizing the window from desktop to tablet view